src/hist/plot.py

```diff
@@ -261,20 +265,27 @@ def plot_pull(
         )
     yerr = np.sqrt(variances)
 
-    if type(func) in [str]:
+    if isinstance(func, str):
         if func == "gaus":
             # gaussian with reasonable initial guesses for parameters
-            constant = ydata.max()
+            constant = float(ydata.max())
```

Comment:
Not always fond of NumPy's choices here. Not sure what a `number[Any]` is. So just forcing it to a Python float.